src/components/RightSidebar/Participants/ParticipantsTab.vue

```diff
@@ -203,18 +206,35 @@ export default {
 
 	beforeMount() {
 		EventBus.$on('route-change', this.abortSearch)
+		EventBus.$on('signaling-users-changed', this.updateUsers)
```

Comment:
Shouldn't `useGetParticipants` handle the update on `'signaling-participant-list-changed'` already ?

Reply:
It doesn't have the required payload: 
https://github.com/nextcloud/spreed/blob/0c7ed6776f1f129e3a1f16794f97c3a1be7f8e85/src/utils/signaling.js#L1514-L1518

Reply:
Fetch requests are mostly in getParticipants composable. Also, there will be duplicated requests when moving to lobby ( request from this method and the other from TopBar's composable).

You need the data only to check if the fetch is needed. However, itis better improvement than only doing from composable because it will not trigger fetch on every user change but only self. 

So I think, this is the best option so far...

Reply:
> Also, there will be duplicated requests when moving to lobby 

There won't be, as TopBar is unmounted at the same time, and won't make any new requests. Even this last one won't be triggered, because composable has an internal check for `isInLobby`
For the `mounted`, we didn't consider that variant before for composable, but it shouldn't be triggered before user is joined conversation => should be fine as-is, i believe